third-party/realdds/src/dds-device-impl.cpp

```diff
@@ -359,7 +370,11 @@ bool dds_device::impl::init()
                     LOG_DEBUG( "... stream '" << stream_name << "' (" << _streams.size() << "/" << n_streams_expected
                                               << ") received with " << profiles.size() << " profiles" );
 
-                    stream->init_profiles( profiles, default_profile_index );
+                    if( _md_supported )
+                    {
+                        auto md_stream_name = stream_name + " metadata";
```

Comment:
If we want to keep it ROS-compatible, can't have spaces in the topic name

Reply:
I used `_` in my branch. E.g.:
`rt/realsense/D435I_012345678_Color`
(can't have more than one `/`, too...)

Reply:
Will change to `_metadata`, other changes will be when merging our changes